context.go

```diff
@@ -45,6 +47,10 @@ func (c *Context) AddNewRelicAttribute(key string, val interface{}) {
 	}
 }
 
+func (c *Context) AppendNewRelicToClient(httpClient *http.Client) *http.Client {
```

Comment:
hmmm ... How about a factory method like NewHttpClient ?

Reply:
Not a fan of this method on the context - given that it is wrapping a call to NewHttpClient, can you remove this method and just use the NewHttpClient function from your code instead? At least until we come up with a better way of dealing with http clients in this framework.

I think in general a few of us need to catch up to discuss how we solve how we want Http Clients work in this framework. Perhaps we can have that discussion soon.

Reply:
Ok, sounds good. Will remove.